virttest/env_process.py

```diff
@@ -58,10 +58,11 @@
 try:
     import PIL.Image
 except ImportError:
-    logging.warning('No python imaging library installed. PPM image '
-                    'conversion to JPEG disabled. In order to enable it, '
-                    'please install python-imaging or the equivalent for your '
-                    'distro.')
+    LOG_UI = logging.getLogger('avocado.app')
+    LOG_UI.warning('No python imaging library installed. PPM image '
```

Comment:
Can you explain what is the reason for choosing the UI output for this? I think it was only previously printed in the log files.

Reply:
Well, I just want to find a suitable logger to handle these messages, so I think highlighting those warnings in the UI is a notification for users. but it's okay for both LOG_UI and LOG_JOB.
Actually, as I discussed with @luckyh offline, a logger provided by VT is a good enhancement to handle some logs, avocado should improve the behavior of handling the RootLogger as well.
Anyway, they are another topic, but it is an optimization indeed that we can consider.

Reply:
I guess it could be useful on the UI side too as long as we make sure that these imports are always needed. There are some libvirt or qemu specific dependencies for instance that some of us don't want to install to begin with and it could be too noisy to look at a warning message like this on each run if you don't have to install the given dependency for your own runs.

Reply:
Thank you @pevogam for your clarification. And after my investigation, these messages will not be saved to the log files if using other loggers, because avocado will only save logging messages after `run_test`, so the messages generated during the import phase are lost. I will fall back to use LOG_UI.